docker/Dockerfile

```diff
@@ -9,12 +9,16 @@ FROM postgres:${PG_VERSION_MAJOR}-bookworm AS builder
 
 ARG PG_VERSION_MAJOR=16
 ARG RUST_VERSION=1.80.0
-ARG PGRX_VERSION=0.11.3
+ARG PGRX_VERSION_PG_SEARCH=0.12.1
```

Comment:
We should combine these two that are the same now and keep the cargo-pgrx install in the builder stage. I'm also wondering whether we should drop pgvectorscale eventually... I'm not sure anyone uses it.